algorithms/1-intro/intro_algo_answers.txt

```diff
@@ -0,0 +1,56 @@
+1.
+  a.)
+  DEF MAKE_AN_OMELETTE
+    HEAT pan on stove
+    CRACK eggs into mixing bowl
+    ADD milk TO bowl
+    REPEAT
+      whisk eggs
+    UNTIL eggs bubbly
+    ADD butter TO pan
+    IF butter completely melted
```

Comment:
They do not go over this, but I do see `WHEN` utilized fairly often to essentially mean:

    REPEAT
        wait
    UNTIL something is true

Such as:

    WHEN something is true THEN
        do this stuff
    ENDWHEN